adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -608,6 +623,7 @@ func createUpstreamTLSContext(upstreamCerts []byte, address *corev3.Address) *tl
 				CipherSuites:              ciphersArray,
 			},
 			TlsCertificates: []*tlsv3.TlsCertificate{tlsCert},
+			// AlpnProtocols:   []string{"h2", "http/1.1"},
```

Comment:
Is there a reason to keep this?

Reply:
forgot to remove it, will remove

Reply:
it is needed to negotiate to h2 with tls. will uncomment this